scripts/dockerhost/start-dockerhosts

```diff
@@ -1,5 +1,56 @@
 #!/bin/bash
 
+# check if all prequisites are installed before proceeding with setup.
+# Pre-requisites:
+# - Ubuntu 14+
+# - Docker 1.6+
+# - nsenter
+# - bridge utils
+# - ovs
+
```

Comment:
@jainvipin 
These pre-requisites are just for the host(server/vm) where docker based setup needs to be brought up. etcd is packaged and run as part of the docker image.